components/soap/src/main/java/org/switchyard/component/soap/composer/SOAPContextMapper.java

```diff
@@ -47,6 +51,12 @@
  */
 public class SOAPContextMapper extends BaseRegexContextMapper<SOAPBindingData> {
```

Comment:
Why is this being done in SOAPContextMapper? Is there any problem to do it in BaseRegexContextMapper like I told you? If there's any reason to do in SOAPContextMapper instead of BaseRegexContextMapper, then you would need to do same thing in all of other context mappers for all bindings.